test/TPP/transform/transform-mlp.mlir

```diff
@@ -104,9 +104,16 @@ transform.sequence failures(propagate) {
     // Detect relus and identity
     %3 = transform.structured.match ops{["func.func"]} in %arg1
     transform.structured.map_linalg_to_tpp %3
+
+    transform.bufferization.one_shot_bufferize %arg1 {
```

Comment:
We should do this to all tests, right?

Reply:
yes. Now we have all the transforms to go from linalg (at tensor) to tpp (at memref).